R/loadWorkbook.R

```diff
@@ -264,7 +265,9 @@ loadWorkbook <- function(file, xlsxFile = NULL, isUnzipped = FALSE) {
     vals <- getNodes(xml = sharedStrings, tagIn = "<si>")
 
     if ("<si><t/></si>" %in% vals) {
-      vals[vals == "<si><t/></si>"] <- "<si><t>NA</t></si>"
+      if (na.convert == TRUE){
```

Comment:
please change this to `if (na.convert) {`

Reply:
done. ✔️

Reply:
thanks